tools/pip/setup.py

```diff
@@ -58,6 +58,10 @@
 elif travis_tag.startswith('patch-'):
     __version__ = os.environ['TRAVIS_TAG'].split('-')[1]
 
+# release tag
+elif is_release:
+    __version__ += 'RELEASE'
```

Comment:
"mxnet-2.0.0RELEASE-py3-none-manylinux2014_x86_64.whl" may not be the correct naming scheme for uploading to Pypi. Also if we install mxnet from pypi and call `mxnet.__version__`, seeing `2.0.0.alphaRELEASE` may be unexpected. Do you think we should generate the wheels to be ready for upload?

Reply:
@leezu thanks for reviewing! I thought we would manually rename the file. In this case I can simply remove the "RELEASE" and then `mxnet-2.0.0-py3-none-manylinux2014_x86_64.whl` should be compliant I think?